context.go

```diff
@@ -126,6 +126,36 @@ func (c *Ctx) LoadProject(path string) (*Project, error) {
 	return p, nil
 }
 
+// resolveProjectRoot evaluates the root directory and does the following:
+//
+// If the passed path is a symlink outside GOPATH to a directory within a
+// GOPATH, the resolved full real path is returned.
+//
+// If the passed path is a symlink within a GOPATH, we return an error.
+//
+// If the passed path isn't a symlink at all, we just pass through.
+func (c *Ctx) resolveProjectRoot(path string) (string, error) {
+	// Determine if this is a symlink
```

Comment:
I had the same thought too, I'll put it back in for a cleaner/quicker passthrough.